docs/basic/cloud-kernel.md

```diff
@@ -0,0 +1,124 @@
+# Using AllStarLink/app_rpt with Cloud Kernels
+
+Many virtual hosts or VPS providers leverage the Debian "cloud kernel"
+version of the Debian Linux OS. These providers include Amazon AWS,
+Microsoft Azure, Linode, Vultr, and DigitalOcean. Using their stock
```

Comment:
The most recent VPS provider reported to have issues was DreamHost.  Do we want to expand the list?